### PR TITLE
Make clippy happy

### PR DESCRIPTION
**Pull request by miDeb**
Closes https://github.com/dotenv-linter/dotenv-linter/issues/402.

#### ✔ Checklist:

- [x] Commit messages have been written in [Conventional Commits](https://www.conventionalcommits.org) format;
- [ ] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);

### CODE REVIEW

File: src/common/file_entry.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 use std::fmt;
 use std::fs;
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
```
Comment (miDeb) [inline src/common/file_entry.rs:3]: It looks like it's the convention to put each import on its own line, should I split this up?

Comment (mgrachev) [reply src/common/file_entry.rs:3]: No, everything is fine 👌

**Review by mgrachev — APPROVED** [review]
@miDeb 👋 Thank you for your contribution 🚀

**Comment by mgrachev** [comment]
@miDeb Thank you for your help 👍

🙏 If it’s not difficult for you, please support the project - star on GitHub ⭐️

**Comment by miDeb** [comment]
Thanks - done!

**Pull request merged**